pylint/checkers/strings.py

```diff
@@ -210,6 +211,12 @@
         "Used when we detect a string that does not have any interpolation variables, "
         "in which case it can be either a normal string without formatting or a bug in the code.",
     ),
+    "W1311": (
+        "The %s syntax imply an f-string but the leading 'f' is missing",
+        "possible-forgotten-f-prefix",
+        "Used when we detect a string that uses '{}' with a local variable inside. "
```

Comment:
Depending on whether we are able to do this, this might need to be "with a local variable or valid expression inside".